modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,58 +25,59 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
 
-NAME                              ENDPOINTS                                                        AGE
-endpoints/ovn-kubernetes-master   10.0.132.11:9102,10.0.151.18:9102,10.0.192.45:9102               57m
-endpoints/ovn-kubernetes-node     10.0.132.11:9105,10.0.143.72:9105,10.0.151.18:9105 + 7 more...   57m
-endpoints/ovnkube-db              10.0.132.11:9642,10.0.151.18:9642,10.0.192.45:9642 + 3 more...   57m
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/ovnkube-control-plane   3/3     3            3           47m
+
+NAME                                               DESIRED   CURRENT   READY   AGE
+replicaset.apps/ovnkube-control-plane-55d6cb89fb   3         3         3       47m
+
+NAME                                     ENDPOINTS                                                AGE
+endpoints/ovn-kubernetes-control-plane   10.0.0.3:9106,10.0.0.4:9106,10.0.0.5:9106                47m
+endpoints/ovn-kubernetes-node            10.0.0.3:9105,10.0.0.4:9105,10.0.0.5:9105 + 15 more...   47m
 
 NAME                                 DATA   AGE
-configmap/control-plane-status       1      55m
-configmap/kube-root-ca.crt           1      57m
-configmap/openshift-service-ca.crt   1      57m
-configmap/ovn-ca                     1      57m
-configmap/ovn-kubernetes-master      0      55m
-configmap/ovnkube-config             1      57m
-configmap/signer-ca                  1      57m
+configmap/control-plane-status       1      46m
+configmap/kube-root-ca.crt           1      47m
+configmap/openshift-service-ca.crt   1      47m
+configmap/ovn-ca                     1      47m
+configmap/ovnkube-config             1      47m
+configmap/signer-ca                  1      47m
 ----
 +
-There are three `ovnkube-masters` that run on the control plane nodes, and two daemon sets used to deploy the `ovnkube-master` and `ovnkube-node` pods.
-There is one `ovnkube-node` pod for each node in the cluster.
-In this example, there are 5, and since there is one `ovnkube-node` per node in the cluster, there are five nodes in the cluster.
-The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations started by online-master and `ovnkube-node`.
-The `ovn-kubernetes-master` `ConfigMap` has the information of the current online master leader.
+There are three `ovnkube-masters` that run on the control plane nodes, and one daemon set used to deploy the `ovnkube-control-plane` and `ovnkube-node` pods.
+There is one ovnkube-node pod for each node in the cluster. In this example, there are six, and since there is one ovnkube-node per node in the cluster, there are six nodes in the cluster.
+The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations started by online leader and `ovnkube-node`.
+The `ovnkube-config` `ConfigMap` has the information of the current online leader pod.
```

Comment:
its missing if its ovnkube-control-plane leader ? i guess? would be good to check on a live cluster.